docs/tutorials/exptime.rst

```diff
@@ -0,0 +1,492 @@
+    import matplotlib.pyplot as plt
+
+    from astroplan import FixedTarget
+    from astroquery.gaia import Gaia
+
+    hatp11 = FixedTarget.from_name('Hat-p-11')
+
+    # width / height of search:
+    width = u.Quantity(1, u.arcmin)
+    height = u.Quantity(1, u.arcmin)
+    search_results = Gaia.query_object_async(coordinate=hatp11.coord, width=width, height=height)
+
+    # the queried star should be the one nearest to the given coordinates
+    search_results.add_index('dist', unique=True)
+    hatp11_info = search_results.loc['dist', min(search_results['dist'])]
+
+    T_eff = round(hatp11_info['teff_val'], -2)  # round to nearest 100 K
+
+    stellar_radius = hatp11_info['radius_val'] * u.R_sun
+    parallax = hatp11_info['parallax'] * u.mas
+    # parallax is given by Gaia in milliarcseconds, so convert to parsec:
+    distance = parallax.to(u.parsec, equivalencies=u.parallax())
+
+    distance_scale = float(stellar_radius / distance) ** 2 / np.pi
+
+    from astropy.io import fits
+
+    flux_url = ('ftp://phoenix.astro.physik.uni-goettingen.de/v2.0/HiResFITS/'
+           'PHOENIX-ACES-AGSS-COND-2011/Z-0.0/lte{T_eff:05d}-{log_g:1.2f}-0.0.PHOENIX-'
+           'ACES-AGSS-COND-2011-HiRes.fits').format(T_eff=int(T_eff), log_g=4.5)
+    wavelength_url = ('ftp://phoenix.astro.physik.uni-goettingen.de/v2.0/HiResFITS/'
+                      'WAVE_PHOENIX-ACES-AGSS-COND-2011.fits')
+
+    flux_of_target = fits.getdata(flux_url) * distance_scale
+    waveset_of_target = fits.getdata(wavelength_url)
+
+    flux_of_target = flux_of_target * (u.erg / u.s / u.cm ** 2 / u.cm)
+    waveset_of_target = waveset_of_target * u.Angstrom
+
+    from astroplan import Observer
+
+    observer = Observer.at_site('apo')
+
+    from synphot.spectrum import SpectralElement
+    from astropy.utils.data import download_file
+    from astroplan import Telescope
+
+    diameter = 3.5 * u.m
+    gain = 1.9 * (u.ct / u.adu)
+
+    # model the bandpass:
+    svo_link = ('http://svo2.cab.inta-csic.es/' +
+               'theory/fps3/fps.php?ID=')
+    filt_path = download_file(svo_link + 'SLOAN/SDSS.rprime_filter')
+    bandpass = SpectralElement.from_file(filt_path)
+
+    telescope = Telescope(diameter, bandpass, gain, ccd_response='default')
+    ccd_response_wls, ccd_response = telescope.ccd_response
+
+    from astroplan import exptime_from_ccd_snr
+
+    snr = 100
+    exptime_from_ccd_snr(snr, waveset_of_target, flux_of_target, observer, telescope)
+
+    from astroplan import skymodel
+
+    skymodel_1pt5airmass = skymodel(airmass=1.5)
+
+    observer.skymodel = skymodel_1pt5airmass
+    # alternatively: >>> observer = Observer.at_site('apo', skymodel=skymodel_1pt5airmass)
+    skymodel_waveset, skymodel_flux = observer.skymodel
+    plt.plot(skymodel_waveset, skymodel_flux)
+    plt.show()
+
+And get the exposure time:
+
+.. code-block:: python
+
+    >>> exptime_from_ccd_snr(snr, waveset_of_target, flux_of_target, observer, telescope)
+    0.0026073144s
```

Comment:
Maybe you could add a small one-line note after this code block which says "Note that the exposure time is slightly longer when accounting for the atmosphere than it was without an atmosphere, due to slight atmospheric extinction in r' the band."